Rakefile

```diff
@@ -63,18 +67,19 @@ namespace :test do
     sh("cd #{integration_dir}/build/ && terraform destroy -force")
   end
 
-  task :destroy_test_environment do
+  task :destroy_test_environment, :namespace do |t, args|
```

Comment:
This looks good.  I confirmed that I can call the **configure** and **destroy** tasks individually with the argument:

```
ubuntu@ubuntu-xenial:~/inspec-aws$ bundle exec rake test:destroy_test_environment[cr-foo]
----> Destroying terraform environment
cd /home/ubuntu/inspec-aws/test/integration/build/ && terraform env select default
Switched to environment "default"!
cd /home/ubuntu/inspec-aws/test/integration/build && terraform env delete cr-foo
Deleted environment "cr-foo"!
```